rules/application/rpc_firewall/rpc_firewall_sasec_recon.yml

```diff
@@ -1,4 +1,4 @@
-title: Remote Schedule Task Lateral Movement via SASec
+title: Recon Activity via SASec
```

Comment:
https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#date
need to update modified

Reply:
Done